includes/options/class-amp-options-manager.php

```diff
@@ -48,6 +49,7 @@ public static function register_settings() {
 
 		add_action( 'update_option_' . self::OPTION_NAME, array( __CLASS__, 'maybe_flush_rewrite_rules' ), 10, 2 );
 		add_action( 'admin_notices', array( __CLASS__, 'persistent_object_caching_notice' ) );
+		add_action( 'admin_notices', array( __CLASS__, 'render_cache_miss_notice' ) );
```

Comment:
@westonruter As we are automatically turning off the response cache, I'm thinking we may want to notify not only on this settings page, but on other pages too.  Why? We can't be certain the admin/developer will come to this page.  At the very least, I'm thinking this notice should be on all AMP settings' pages.  What do you think?

Reply:
What if we limit it to the general settings page but add an indicator to the admin menu item so that the user knows there has something changed on this page?

Reply:
@westonruter I like that approach in general.  I think it spans more than just this PR and edge case.  Thinking about, it seems we could have a general indicator to alert of something that needs attention.  That indicator could link to the plugin's settings page where more information lives.

If we go for that approach, I think that's a separate feature, issue, and PR.